src/Symfony/Component/VarDumper/composer.json

```diff
@@ -22,6 +22,7 @@
     "require-dev": {
         "ext-iconv": "*",
         "symfony/console": "^5.4|^6.0|^7.0",
+        "symfony/error-handler": "^6.3|^7.0",
```

Comment:
Why do you need it?

Reply:
for the following line in ExceptionCasterTest:
`-dataRepresentation: ? Symfony\Component\VarDumper\Cloner\Data`